### ISSUE TITLE
initramfs image generated is bigger than dracut -f 

### ISSUE THREAD

**Issue by dougsland** [containers/initoverlayfs#2]
The initramfs generated by `dracut -f` is smaller than the one generated by initoverlayfs.

**Info**:
``` bash
# cat /etc/fedora-release
Fedora release 39 (Thirty Nine)

# rpm -qa | grep -i initoverlayfs
initoverlayfs-0.96-1.fc39.x86_64
```

**Generating the image and comparing the size**

Generating using raw dracut:
``` bash
# dracut -f
# du -sh /boot/initramfs-6.5.5-300.fc39.x86_64.img
34M	/boot/initramfs-6.5.5-300.fc39.x86_64.img
```

Generating using initoverlayfs:
``` bash
#  /usr/bin/initoverlayfs-install
# du -sh /boot/initramfs-6.5.5-300.fc39.x86_64.img
42M	/boot/initramfs-6.5.5-300.fc39.x86_64.img
```

**Comment by dougsland** [comment]
I will start comparing the two images:

```
# sudo lsinitrd ./42M-initramfs-6.5.5-300.fc39.x86_64.img &> 42m.txt
# sudo lsinitrd ./34M-initramfs-6.5.5-300.fc39.x86_64.img &> 34m.txt
#  diff -ruN 34m.txt 42m.txt
```

**Comment by dougsland** [comment]
In the diff I noticed it included **AuthenticAMD.bin** which in dracut -f is not included:
```
+-rw-r--r--   1 root     root        77182 Oct  4 20:00 kernel/x86/microcode/AuthenticAMD.bin
```

Just for reference, here is the intel cpu:
```
[root@dell730 initoverlayfs]# cat /proc/cpuinfo  | grep -i intel|more
vendor_id	: GenuineIntel
model name	: Intel(R) Xeon(R) CPU E5-2687W v3 @ 3.10GHz
vendor_id	: GenuineIntel
```

**Comment by ericcurtin** [comment]
Fix submitted:

https://github.com/containers/initoverlayfs/pull/3

Feel free to test by copying this change into:

`/usr/bin/initoverlayfs-install`

doing a:

`rm -f /etc/initoverlayfs.conf`

and it should be smaller again.

**Comment by dougsland** [comment]
For me it worked, I would just merge it.

**Comment by ericcurtin** [comment]
Thanks for looking at this!

**Comment by dougsland** [comment]
resolved via #3
